lib/views/pages/account_settings/account_settings.dart

```diff
@@ -39,11 +40,12 @@ class AccountSettings extends HookConsumerWidget {
                   mainAxisAlignment: MainAxisAlignment.center,
                   children: <Widget>[
                     IconButton(
-                        icon: const Icon(Icons.edit),
-                        onPressed: () {},
-                        color: Colors.transparent),
+                      icon: const Icon(Icons.edit),
+                      onPressed: () {},
+                      color: Colors.transparent,
+                    ),
                     Text(
-                      first_name + ' ' + last_name,
+                      firstName + ' ' + lastName,
```

Comment:
use string interpolation instead of concatenation ([reference](https://dart-lang.github.io/linter/lints/prefer_interpolation_to_compose_strings.html))